src/camera_streamer/camera_streamer/stream_fuser.py

```diff
@@ -0,0 +1,112 @@
+import rclpy
+from rclpy.node import Node
+from rclpy.action import ActionServer, GoalResponse
+from rclpy.executors import MultiThreadedExecutor
+from sensor_msgs.msg import Image
+from action_interfaces.action import Fuser
+import re
+class ImageSelectorNode(Node):
+    def __init__(self):
+        super().__init__('image_selector')
+        self.get_logger().info('ImageSelector node starting up')
+        self.topic_pattern = re.compile(r'^/usbcam_image_[0-9A-Za-z_~{}]+$')
```

Comment:
feel like this regex could be simpler but i honestly don't know what pattern youre looking for.